bot/answers.py

```diff
@@ -0,0 +1,187 @@
+'''
+File with all buttons and keyboards for bot
+'''
+
+# Matkup
+from telebot.types import ReplyKeyboardMarkup, KeyboardButton, \
+InlineKeyboardButton, InlineKeyboardMarkup
+
+
+# Format functions
+def buttonformcourierstatus(text, txtbtn, call):
```

Comment:
Please read the style guide and fix naming https://github.com/google/styleguide/blob/gh-pages/pyguide.md#316-naming